File: c++/CFLOW008.cc
#include <iostream>
using namespace std;

int main()
{
    int a, b;
    cin >> a;
    while (a > 0)
    {
        cin >> b;
        if (b < 10)
        {
            cout << "What an obedient servant you are!" << endl;
        }
        else
        {
            cout << "-1" << endl;
        }
        a -= 1;
    }
}

File: scala/CFLOW008.scala
object CFLOW008 {
  def main(args: Array[String]): Unit = {
    var a = scala.io.StdIn.readInt()
    while (a > 0) {
      var b = scala.io.StdIn.readInt()
      if (b < 10) {
        println("What an obedient servant you are!")
      } else {
        println("-1")
      }
      a -= 1
    }
  }
}